apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php

```diff
@@ -203,11 +203,19 @@ public function output_template_content( $template_type ) {
 
 		// 10 priority
 		$content = wptexturize( $content );
-		// @todo maybe look at WPCOM_Responsive_Images for WPCom responsive image handling
 
 		// 11 priority
 		$content = do_shortcode( $content );
 
+		$content = prepend_attachment( $content );
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM && class_exists( '\WPCOM_Responsive_Images' ) ) {
```

Comment:
So far we have tried to avoid adding any Dotcom specific code in the FSE plugin. As a non-blocking suggestion: we could add a filter here and modify the content from dotcom side in our mu-plugin code.

Reply:
Oooh, very good point!
It's an easy enough change, I'll do it right now!